migration.go

```diff
@@ -14,7 +14,7 @@ import (
 
 type MigrationRecord struct {
 	VersionId int64
-	TStamp    time.Time
+	TStamp    string
```

Comment:
@VojtechVitek the proposed approach also breaks the API. After some more investigation it turns out that for MySQL driver`?parseTime=true` must be set. 

I'm closing the PR, however, it would be good if you could link somewhere that in order to use MySQL one needs to set https://github.com/go-sql-driver/mysql#parsetime.